pkg/cluster/nodes/create.go

```diff
@@ -146,6 +146,10 @@ func createNode(name, image, clusterLabel, role string, mounts []cri.Mount, port
 		"--tmpfs", "/run", // systemd wants a writable /run
 		// some k8s things want /lib/modules
 		"-v", "/lib/modules:/lib/modules:ro",
+		// ensure pods etc. are not on container filesystem
+		// TODO: we could do this in the image instead
+		// However this would leave old images with this issue
+		"-v", "/var/lib/kubelet",
```

Comment:
hm, this would expose the kind nodes to the host /var/lib/kubelet that may already contain kubelet data, which may result in:
- the kubelet on the node re-using the existing kubelet certificates
- the kubelet on the node stoping kubelet data on the host

is there a link to the original problem?

Reply:
no, that's not what this does. this is an anonymous volume, not a bind mount.

Reply:
https://success.docker.com/article/different-types-of-volumes

Reply:
my mistake.

Reply:
this flag format does a lot of in-band signaling :sweat_smile: 
soon moving this to higher level logic... going to need to re-work that a bit now :upside_down_face: